### DISCUSSION TITLE
Way to stop dbus from listening to abstract sockets?

### DISCUSSION THREAD

**Discussion by matthew-cline** [netblue30/firejail#4272] [Q&A]
Is there any way to make it so that dbus daemons don't listen to abstract UNIX sockets?  Poking around the documentation and doing some web searches hasn't shown anything so far.

**Comment by ghost** [comment]
There are several places where one can find hints on how to deal with abstract UNIX sockets. Try https://github.com/netblue30/firejail/search?q=abstract+socket and have a look at the code results.
